db/cms_fixtures/la2050/pages/home/goals/content.haml

```diff
@@ -0,0 +1,35 @@
+.padded-section
+  .container
+    .col-sm-8.col-sm-offset-2.text-center
+      %h1 Goals
+      %p.prose.primary
+        In order to track and drive progress, Angelenos must first be united by a shared vision of success for the region.
+        In 2014, we launched #LA2050Listens - a series of community conversations - alongside expert roundtables to create a shared vision for the future of Los Angeles. Those conversations shaped the LA2050 Goals and Metrics which you can learn about below.
+      %button.bigger LEARN MORE ABOUT #LA2050Listens
```

Comment:
The button class has `text-transform:uppercase;` on it, so everything will already be rendered in uppercase. Might change this to regular casing for readability.